### PR TITLE
WRO-12454 Fixed deprecation of register.js

### PR DESCRIPTION
**Pull request by andreitirlalge**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
register.js is deprecated error in the console.
error1

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Addons(actions, controls, docs and toolbar) we're being imported from '@storybook/addon-controls' using '/register'. By following this path I found the source file that is throwing this error inside 'node_modules/@storybook/addon-actions/register.js'. Inside this file there is also an import for '/manager'. By changing the addons (actions, controls, docs and toolbar) import from '/register' to '/manager' the console is no longer throwing the register.js warning. 
We've also tried different approaches but this is the only one that worked so far as the link from the warning does not really present other viable solutions.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#deprecated-registerjs
WRO-12454

### Comments
Enact-DCO-1.0-Signed-off-by: Andrei Tirla [andrei.tirla@lgepartner.com](mailto:andrei.tirla@lgepartner.com)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ## Unreleased
 
 * Fixed `showName` warning by changing `title` property to `name` value in the controls addon.
+* Changed addons import module from `register` to `manager` to all addons.
```
Comment (daniel-stoian-lgp) [inline CHANGELOG.md:4]:
```suggestion
* Changed addons import module from `register` to `manager` for all addons.
```

**Review by adrian-cocoara-lgp — APPROVED** [review]
LGTM

**Review by MikyungKim — COMMENTED** [review]
What do you think about renaming `register.js` to `manager.js`?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ## Unreleased
 
 * Fixed `showName` warning by changing `title` property to `name` value in the controls addon.
+* Changed addons import module from `register` to `manager` for all addons.
```
Comment (MikyungKim) [inline CHANGELOG.md:4]: Please keep in mind that the newer change log should be on top of the existing one.

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**